gerenciador-de-tarefas/src/tests/gerenciadordetarefas/GerenciadorTest.java

```diff
@@ -0,0 +1,127 @@
+package gerenciadordetarefas;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GerenciadorTest {
+
+    private Gerenciador gerenciador;
+
+    private Tarefa tarefaExemplo;
+
+    @BeforeEach
+    void setUp(){
+        this.gerenciador = new Gerenciador();
+        this.tarefaExemplo = new Tarefa(0, "Tarefa 1", "Essa é a tarefa 1", LocalDate.now(), Prioridade.BAIXA);
+    }
+
+    void adicionaTarefaExemplo() {
```

Comment:
Esse método poderia ter entrado dentro do setUp(), evitando ser chamado explicaitamente no início de todos os métodos de teste

Reply:
Boa! Eu tinha pensando inicialmente em não fazer isso, porque no primeiro caso de teste não uso esse método. O primeiro caso verifica se um novo gerenciador é criado com uma lista vazia de tarefas. Porém, consegui facilmente resolver esse problema criando um novo objeto gerenciador. Obrigado pela sugestão 😄